fix-code-7/main.cpp

```diff
@@ -9,7 +9,7 @@ int main() {
   int average_pig_weight = 150;
   int average_cow_weight = 300;
 
-  int total_weight = num_pigs * average_cow_weight + num_cows * average_cow_weight;
+  int total_weight = num_pigs * average_pig_weight + num_cows * average_cow_weight;
```

Comment:
While not required, I do think that if you put ( ) around "num_pigs * average_pig_weight" and again around "num_cows * average_cow_weight" it would make it easier for humans to read.